src/test/java/com/google/jenkins/plugins/credentials/oauth/GoogleRobotCredentialsTest.java

```diff
@@ -48,6 +50,8 @@ public class GoogleRobotCredentialsTest {
   // Allow for testing using JUnit4, instead of JUnit3.
   @Rule public JenkinsRule jenkins = new JenkinsRule();
 
+  private static final CredentialsScope CREDENTIALS_SCOPE = CredentialsScope.GLOBAL;
```

Comment:
nit: probably redundant as we creating new constant to reference another constant.